EventsExpress.Core/Services/EventScheduleService.cs

```diff
@@ -105,7 +103,7 @@ public async Task<Guid> CancelNextEvent(Guid eventId)
             return await Edit(eventScheduleDTO);
         }
 
-        private UserDto CurrentUser() =>
-           _authService.GetCurrentUser(_httpContextAccessor.HttpContext.User);
+        private Guid CurrentUserId() =>
```

Comment:
Could we use the new SecurityContext here? It does the same thing but hasn't a lot of dependencies (as Auth service have)

Reply:
We can but then will need to rewrite half of project to change all this stuff(AuthService to SecurityContext).
But still it authService uses SecurityContext to get userId